app/main.js

```diff
@@ -5,6 +5,7 @@ var exported = {
   'child_process': null,
   'uuid': require('uuid'),
   'flowhub-registry': require('flowhub-registry'),
+  'fbp-protocol-client': require('fbp-protocol-client'),
```

Comment:
Who owns the connection to the runtime? What is the lifetime? How do code which needs a reference get it?